software-containers/README.md

```diff
@@ -16,3 +16,18 @@ group:      sss files
 
 ### xterm
 `xterm` must be installed on the container
+
+### ENTRYPOINT
+Use ENTRYPOINT to specify an executable to be run, for example a startup script which can execute several tasks when the container starts up.
+
+### CMD
+Use CMD to specify a command to be executed at the end of the Dockerfile, "xterm".
+
+### desktop
+Each container is associated with a desktop file, for example terminal.desktop, which includes the path to the shell script, for example terminal.sh, which initializes and starts up the container.
+
```

Comment:
This 'desktop' section is part of creating the menu for ARCADE.  Users won't do this so please remove.

Reply:
Okay, will delete it.